string rank
string answerText
string linkText
string linkUrl
